core/src/main/java/org/apache/iceberg/BaseMetastoreCatalog.java

```diff
@@ -238,4 +260,8 @@ protected static String fullTableName(String catalogName, TableIdentifier identi
 
     return sb.toString();
   }
+
+  protected Map<String, String> properties() {
```

Comment:
This should be located with the other instance methods, not at the end after the builder and static methods.